exporters/otlp/common/build.gradle.kts

```diff
@@ -29,6 +29,7 @@ dependencies {
 
   testImplementation("com.fasterxml.jackson.core:jackson-databind")
   testImplementation("com.google.protobuf:protobuf-java-util")
+  testImplementation("com.google.guava:guava")
```

Comment:
Guava switched to a runtime dependency in this version. Test code was relying on it being an API dependency. This restores it for compilation.